src/aspire/source/coordinates.py

```diff
@@ -168,8 +168,8 @@ def _box_coord_from_center(center, particle_size):
         return [
             # centers may be represented as floats in STAR files
             # chop off the non integer part to account for this
-            int(x) - r,
-            int(y) - r,
+            int(float(x)) - r,
```

Comment:
There's no good reason for accepting strings as inputs here. Move the casts upstream:
https://github.com/ComputationalCryoEM/ASPIRE-Python/blob/11c3dcaeae830b86f0a491a97c7ba91e980c7695/src/aspire/source/coordinates.py#L200
https://github.com/ComputationalCryoEM/ASPIRE-Python/blob/11c3dcaeae830b86f0a491a97c7ba91e980c7695/src/aspire/source/coordinates.py#L534